### PR TITLE
ClusterLoader - Adding TestSuite api

### PR DESCRIPTION
**Pull request by krzysied**
ref #641

### CODE REVIEW

File: clusterloader2/api/types.go [hunk 1]
```diff
@@ -20,6 +20,21 @@ import (
...
 
+// TestSuite defines list of test scenarios to be run.
+type TestSuite []TestScenario
+
+// TestScenario defines customized test to be run.
+type TestScenario struct {
+	// Identifier is a unique test scenario name across test suite.
+	Identifier string `json: identifier`
+	// ConfigPath defines path to the config file that satisfy already
+	// existing cluster loader API.
```
Comment (wojtek-t) [inline clusterloader2/api/types.go]: What is "existing cluster loader API"?

It should mention something like that this file contains a single Config.

Comment (krzysied) [reply clusterloader2/api/types.go]: Done.

File: clusterloader2/api/types.go [hunk 2]
```diff
@@ -20,6 +20,20 @@ import (
...
 )
 
+// TestSuite defines list of test scenarios to be run.
+type TestSuite []TestScenario
+
+// TestScenario defines customized test to be run.
+type TestScenario struct {
+	// Identifier is a unique test scenario name across test suite.
+	Identifier string `json: identifier`
+	// ConfigPath defines path to the single test config file.
```
Comment (wojtek-t) [inline clusterloader2/api/types.go]: .. to the file containing a single Config definition.

maybe? or sth like that?

Comment (krzysied) [reply clusterloader2/api/types.go]: Done.

**Pull request merged**